fastlane/lib/fastlane/actions/create_xcframework.rb

```diff
@@ -24,6 +31,27 @@ def self.run(params)
         sh(create_command)
       end
 
+      def self.normalized_artifact_info(artifacts_with_info, valid_info)
+        case artifacts_with_info
+        when Array
+          artifacts_with_info.map { |artifact| [artifact, {}] }.to_h
+        when Hash
+          artifacts_with_info.transform_values { |artifact_info| artifact_info.transform_keys(&:to_sym).slice(*valid_info) }
```

Comment:
Can we add a comment with an example of what this line is doing? I _think_ I get it but I might not if I come back to this 😛

Reply:
Sure

Reply:
Fixed in 14d240e. Please re-open if the comment is not clear enough.

Reply:
I also included an example in 715d295